ui/ssb-msg.js

```diff
@@ -88,6 +104,31 @@ Vue.component('ssb-msg', {
         else
           this.msg = { key: this.msg.key, value: msgValue }
       })
+    },
+    react: function(emoji) {
+      if(emoji == 'Unlike') {
+	this.myReactions = [];
```

Comment:
can you check indention here and remove the ; ? So it conforms with the rest of the code :-)